CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.4.1 (unreleased)
+## 1.4.1 (2019-08-01)
+
+This maintenance release contains bugfixes since the 1.4.0 release.
```

Comment:
This should change to our new style: 

This maintenance release contains bugfixes since the 1.4.0 release. We deem it low-to-moderate priority for upgrading.

In particular, the fixes contained in this maintenance release do not address any security vulnerabilities, while the only one affecting system stability are related to uncommon queries and prepared statements. More details below.

Reply:
just in case sven copies this in - the only one affecting system stability "is" related to uncommon queries and prepared statements.